sql/core/src/main/scala/org/apache/spark/sql/SparkSession.scala

```diff
@@ -109,7 +109,7 @@ class SparkSession private(
    * A wrapped version of this session in the form of a [[SQLContext]], for backward compatibility.
    */
   @transient
-  private[sql] val sqlContext: SQLContext = new SQLContext(this)
+  private[spark] val sqlContext: SQLContext = new SQLContext(this)
```

Comment:
@rxin Is that OK? Here, the reason why I changed is for the call in https://github.com/gatorsmile/spark/blob/7dcaaa4539324d596c290ca189adc2e65d77bb0c/mllib/src/main/scala/org/apache/spark/ml/util/ReadWrite.scala#L76